src/main/java/com/sparta/popupstore/domain/popupstore/dto/request/PopupStoreCreateRequestDto.java

```diff
@@ -0,0 +1,20 @@
+package com.sparta.popupstore.domain.popupstore.dto.request;
+
+import lombok.Getter;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+@Getter
+public class PopupStoreCreateRequestDto {
+    private String name;
+    private String content;
+    private String imageUrl;
+    private String price;
```

Comment:
```suggestion
    private int price;
```
String으로 입력받을 필요가 없으면 int 로 입력 받는게 좋을 것 같습니다.

Reply:
변경 완료했습니다.